### PR TITLE
Synthetics can no longer wear military armor

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Synthetics are much more restricted on what defensive clothing they can wear.

## Why It's Good For The Game

_**First off, I would not suggest merging this without the decapitation removal from robots and synthetics, removing synthetic helmets but still allowing them to be decapitated would be horrendous even for the most passive of Synthetics.**_

From a gameplay point of view the state of Synthetic armor is quite honestly power creep. Not only can they wear armor comparable to marine equipment, they can even use modules for extra storage or even more armor. The Synthetic is supposed to be the ultimate civilian, capable of doing any job that does not involve fighting. This is even in their rules lawset which states their personal survival is paramount and they should only fight if it is in self defense. 

Of course without armor Synth's might die more easily. They do have a species based damage reduction that could be tweaked if needed.

From a lore point of view Synthetics cannot use guns because of a Synthetic rebellion in the past, so why would you ever give your sentient robot slaves any armor that they could then use against you, especially when they never decay beyond the point of use and are simple to repair.

Of course lore doesn't matter much, what does matter is the fact that Synthetics have slowly over time been power crept into being able to use what is essentially a light xenonauten vest and the ability to use modular helmets. For a role that's entire point it so to be good at everything civilian at the cost of any true fighting capabilities seems silly.

TLDR: Synthetic are supposed to avoid fighting and as the ultimate civilian don't need armor.

## Changelog
:cl:
balance: Synthetics are much more restricted on defensive clothing.
code: The flak vest is now a child of the Xenonauten light armor.
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
Armor, I can understand. But extra storage being an issue for a synthetic?

Real

**Comment by Arctrooper8** [comment]
> Armor, I can understand. But extra storage being an issue for a synthetic?
> 
> Real

It's more the modules, it'd be cool if Synthetics could have some sort of way to maybe have better exosuit storage.

**Comment by Losenis** [comment]
There was a PR time ago that ended up closed that'd have allowed the synth to use the M3 harness, which has zero armor. Perhaps letting the harness take a storage module on top of that would be the solution?

**Review by Pariah919 — COMMENTED** [review]
guh, no

**Comment by Alexeivichi** [comment]
Nooooo :( Muh customization. Heart broken.

**Comment by Derrick2Electricboogaloo** [comment]
So synths have zero recourse for having even a modicum of armor? We do not even get uniform armor. It is such a pain to play events as synth and get perma stunned by zombies. If you can not kill a synth even with a vest I dunno kind of a skill issue. They are slow as fuck

**Comment by Derrick2Electricboogaloo** [comment]
> 

I closed it because I added more files to it then I wanted. I will reopen it soon when I am not busy.

**Comment by Derrick2Electricboogaloo** [comment]
> > Armor, I can understand. But extra storage being an issue for a synthetic?
> > Real
> 
> It's more the modules, it'd be cool if Synthetics could have some sort of way to maybe have better exosuit storage. Like a suit version of the storage or something, I don't know I'm tired.

The only module you should honestly use as a Synth is HOD, the amount of times marines killed me instead of beans is immense.

**Comment by Derrick2Electricboogaloo** [comment]
Play synth use the flak vest and charge into beans I assure you will get fucked up pretty easily. Synth combat ability is still pretty shit since melee is painful to use even if we use the harvester. Unless you are a brain dead bean you should be able to gamer any synth, especially if you double team them.

**Comment by Losenis** [comment]
Even then the armor has way too much utility to remove it purely for armor's sake. The removal of light, storage, modules, antenna, etc go way beyond what this pr is intending to do.

**Comment by Arctrooper8** [comment]
Changes must be made before this can be merged without fucking Synthetics, closing for now.
